requirements.txt

```diff
@@ -11,3 +11,4 @@ chardet>=3.0.4
 fastavro>=1.0.0.post1
 python-snappy>=0.5.4
 h5py>=2.10.0
+tqdm>=4.59.0
```

Comment:
I don't necessarily like adding another requirement. Also, this will be executed over SSH and non-standard terminal environments.

Reply:
Plz remove requirement

Reply:
Removed the requirement
This makes sense as I'm not sure what would happen if tqdm tried to output to a non-standard terminal environment, but it does take away from a user who is unaware of this feature and installs to a virtual environment (since it probably wouldn't have tqdm).